core-bundle/assets/controllers/toggle-state-controller.js

```diff
@@ -0,0 +1,86 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static targets = ['controller', 'controls'];
+
+    static values = {
+        controlsClass: {
```

Comment:
this should use Stimulus CSS class (https://stimulus.hotwired.dev/reference/css-classes)

Reply:
Added in 34c3e67f7167a9266068c10c3396ae4d996772e7